pkgs/tools/audio/beets/builtin-plugins.nix

```diff
@@ -10,6 +10,11 @@
   ...
 }:
 {
+  _typing = {
+    testPaths = [ ];
+    enable = false;
```

Comment:
Maybe we can enable this depending on the beets derivation that triggers the evaluation of this file? It'd require another argument the file will accept..

Reply:
I do sorta by setting enable and builtin to true in the `unstable` derivation.